docs/serving/samples/hello-world/helloworld-java-spring/README.md

```diff
@@ -27,12 +27,12 @@ knative-docs/serving/samples/hello-world/helloworld-java-spring
    commands:
 
    ```shell
-   curl https://start.spring.io/starter.zip \
+   $ curl https://start.spring.io/starter.zip \
```

Comment:
@janitha09 Our style is to not show command prompts like `$` before commands, so let's take these out for consistency with the rest of the docs.